media/js/src/simulations/simulation2/components/multipleChoiceQuestion2.jsx

```diff
@@ -47,20 +45,31 @@ export const MultipleChoiceQuestion2 = ({isSubmitted, setIsSubmitted, takeaways,
     }, [results]);
 
     useEffect(() => {
```

Comment:
This useEffect and the one above are both listening to Results. Can they be combined?